apps/site/assets/css/_event-details.scss

```diff
@@ -16,4 +16,30 @@
   .c-paragraph {
     padding-bottom: $base-spacing * 2;
   }
+
+  .m-event-section {
+    padding-bottom: $base-spacing-lg;
+  }
+
+  .m-event__registration {
+    display: flex;
+    justify-content: space-between;
+    
+    &-header {
+      flex: 0 1 40%;
+      margin: 0;
+      align-self: center;
+    }
+
+    .btn {
+      width: 100%;
+    }
```

Comment:
Bootstrap has a helper class for this! If you add the `btn-block` class to the button it'll do exactly this for you! 🥳